src/org/labkey/test/components/QueryMetadataEditorPage.java

```diff
@@ -0,0 +1,91 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class QueryMetadataEditorPage extends DomainDesignerPage
+{
+    public QueryMetadataEditorPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    @Override
+    public void clickFinish()
+    {
+        scrollIntoView(elementCache().finishButton());
+        shortWait().until(ExpectedConditions.elementToBeClickable(elementCache().finishButton()));
+        click(Locator.button(elementCache().finishButton().getText()));
```

Comment:
Is there any sort of success indicator to wait for? Just clicking and continuing can cause tests to ignore errors.

Reply:
'Save successful' is a text response after clicking Save on Query Metadata Editor.